src/main/java/me/anxuiz/settings/bukkit/plugin/gui/clickables/Click.java

```diff
@@ -0,0 +1,11 @@
+package me.anxuiz.settings.bukkit.plugin.gui.clickables;
+
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.event.inventory.InventoryCloseEvent;
+
+public interface Click {
```

Comment:
What is the purpose of this interface? Nothing calls these methods, they are just overridden as `@EventHandler`s. And why are they named register and unregister? Shouldn't they be click and close?